wolfssl/internal.h

```diff
@@ -1842,9 +1842,18 @@ WOLFSSL_LOCAL int  SetCipherList(WOLFSSL_CTX*, Suites*, const char* list);
 #endif /* WOLFSSL_DTLS_EXPORT_TYPES */
 
 
+#if defined(OPENSSL_ALL) || defined(WOLFSSL_QT)
+#define UINT8_SZ 255
```

Comment:
Naming of macro here could be adjusted. There is potential to clash with a UINT8_SZ macro set for the size of an unsigned byte. Maybe use MAX_SUITE_NAME from wolfssl/internal.h or MAX_CIPHERNAME_SZ  from wolfssl/internal.h or something equivalent.

Reply:
I went with `MAX_CIPHERNAME_SZ`